src/release/issue-triaging.md

```diff
@@ -145,6 +145,15 @@ There are many different labels that can be applied to issues.
   - An FCP will also cause a relnotes issue to be created, if it's started on an issue.
 - [`metabug`]: Tracks other bugs.
 
+### Creating labels
+
+<div class="warning">
+Triagebot needs to support `@rustbot label: xxx.` usages terminated with a period (as inline invocation), so the label name must not contain periods (`.`).
```

Comment:
I think we are using any punctuation (and spaces, of course) as token separators, so I'd be more generic about what not to use in labels.

Reply:
Wait. We have a few labels with spaces in them, right...?

Reply:
Maybe rephrase it as "label name must be alphanumeric + hyphen (`-`) + underscore (`_`) only"? Is that more accurate?

Reply:
> Wait. We have a few labels with spaces in them, right...?

I *do* hope we don't :-) that would break the [tokenizer parser](https://github.com/rust-lang/triagebot/blob/master/parser/src/token.rs#L194-L195).

```
$ for i in `seq 1 10`; do curl -Ss -H "Authorization: Bearer $GITHUB_TOKEN" "https://api.github.com/repos/rust-lang/rust/labels?per_page=100&page=$i" | jq '.[].name' | grep " " && echo "AAAAHHHH found a space" ; done
$ ¯\_(ツ)_/¯
```

Reply:
I double-checked, I don't think we have any atm (phew)

Reply:
Fixed the wording to reflect the constraints, lmw what u think.